kytos/core/interface.py

```diff
@@ -464,15 +473,22 @@ def _make_tags_available(
             new_tags, conflict = range_addition(tags, available_tags)
             self.available_tags[tag_type] = new_tags
             return conflict
-        result = self._add_tags(tags, tag_type)
-        if result is False:
-            return [tags]
-        return []
+        if isinstance(tags, list):
+            result = self._add_tags(tags, tag_type)
+            if result is False:
+                return [tags]
+        if isinstance(tags, str):
+            if tags not in self.special_available_tags[tag_type]:
```

Comment:
Notice that this check would allow to add invalid arbitrary string tags, let's say `"foo", "bar"`. In the same way that for `vlan` type we're only allowing values based on tag_ranges, I believe we should have an equivalent `special_tag_ranges` or something along this lines that could be configurable too, but by default only having `["vlan", "untagged"]`, see what I mean?

Reply:
Added `special_tag_ranges` and its modification method. Unit tests are missing.